components/grids.css

```diff
@@ -0,0 +1,19 @@
+.grid-50-50{
+    display: grid;
+    grid-template-columns: 50% 50%;
+}
+.grid-30-70{
+    display: grid;
+    grid-template-columns: 30% 70%;
+}
+.grid-70-30{
+    display: grid;
+    grid-template-columns: 70% 30%;
+}
+.grid-item-1{
+    background-color: #F7F7DA;
```

Comment:
use variables for colors!

Reply:
I won't reuse these variables, therefore I used the actual code